src/SWP/SuperdeskBridgeBundle/Resources/config/config.yml

```diff
@@ -0,0 +1,4 @@
+parameters:
```

Comment:
those are this same as default values in Configuration - so it's not needed. From other side - why we need internall config.yml? We can'd relly on configuration from app? (and provide documnetation for app config.yml) ?